spec/test.spec.js

```diff
@@ -123,6 +123,27 @@ describe('S3Adapter tests', () => {
       expect(options.bucketPrefix).toEqual('test/');
     });
 
+    it('should accept options and overrides as an option in args', () => {
+      var confObj = { bucketPrefix: 'test/', bucket: 'bucket-1', secretKey: 'secret-1', accessKey: 'key-1' ,s3overrides: { secretAccessKey: 'secret-2', accessKeyId: 'key-2', params: { Bucket: 'bucket-2' }} };
```

Comment:
nit: add a space after the comma

it should be 
`, s3over....`
not 
`,s3over...`